code/modules/modular_computers/computers/machinery/console_presets.dm

```diff
@@ -84,6 +84,18 @@
 	hard_drive.store_file(new/datum/computer_file/program/job_management())
 	hard_drive.store_file(new/datum/computer_file/program/crew_manifest())
 
+/obj/machinery/modular_computer/console/preset/id/centcom
+	desc = "A stationary computer. This one comes preloaded with CentCom identification modification programs."
+
+/obj/machinery/modular_computer/console/preset/id/centcom/install_programs()
+	var/obj/item/computer_hardware/hard_drive/hard_drive = cpu.all_components[MC_HDD]
+	var/datum/computer_file/program/card_mod/card_mod_centcom = new/datum/computer_file/program/card_mod()
+	card_mod_centcom.is_centcom = TRUE
+	hard_drive.store_file(new/datum/computer_file/program/chatclient())
+	hard_drive.store_file(card_mod_centcom)
+	hard_drive.store_file(new/datum/computer_file/program/job_management())
+	hard_drive.store_file(new/datum/computer_file/program/crew_manifest())
```

Comment:
```suggestion
	var/datum/computer_file/program/card_mod/card_mod_centcom = new /datum/computer_file/program/card_mod()
	card_mod_centcom.is_centcom = TRUE
	hard_drive.store_file(new /datum/computer_file/program/chatclient())
	hard_drive.store_file(card_mod_centcom)
	hard_drive.store_file(new /datum/computer_file/program/job_management())
	hard_drive.store_file(new /datum/computer_file/program/crew_manifest())
```
Try to keep a separation between these two so we have an easier time searching in the future.